src/content/docs/codestream/how-use-codestream/my-organization.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Manage my organization on CodeStream
```

Comment:
I'm very torn! Everywhere except in this title, you initial cap my organization. I hate to say it, but I'd capitalize it to match the UI.  (I thought about making it "your organization" but then that opens up other questions.